plugins/modules/proxmox.py

```diff
@@ -415,11 +424,23 @@ def is_template_container(self, node, vmid):
         return config['template']
 
     def create_instance(self, vmid, node, disk, storage, cpus, memory, swap, timeout, clone, **kwargs):
+        # Available only in PVE 7
+        only_v7 = ['tags']
+
         proxmox_node = self.proxmox_api.nodes(node)
 
         # Remove all empty kwarg entries
         kwargs = dict((k, v) for k, v in kwargs.items() if v is not None)
 
+        version = self.version()
+        pve_major_version = 3 if version < LooseVersion('4.0') else version.version[0]
+
+        # The features work only on PVE 7
+        if pve_major_version < 7:
+            for p in only_v7:
+                if p in kwargs:
+                    del kwargs[p]
```

Comment:
I think the module should check the value of `kwargs[p]` first, and fail if it is not `None` so that the user knows that they cannot use this option.

Reply:
I mimicked the behavior of `proxmox_kvm` for this part, which is silently ignoring unsupported parameters. If I change it here, we should probably update `proxmox_kvm` too, for consistency

Reply:
Please change this module, but not the other one. That would be a breaking change and cannot be merged without a deprecation period. (If you want you can start the deprecation process, but please not in this PR.)

Reply:
Done :)